fontbe/src/orchestration.rs

```diff
@@ -117,54 +122,85 @@ impl From<WorkId> for AnyWorkId {
     }
 }
 
+impl From<&FeWorkIdentifier> for AnyWorkId {
```

Comment:
I imagine this is just being moved around? In any case just noticing it now, but I would _generally_ avoid doing `impl From<&T>` if you are just cloning in your impl; it's preferable to just `impl From<T>` and then force the caller to be explicit about the clone.

This is a broader idea in rust of avoiding writing methods that ask for things to be passed by value, but then clone internally; the caller can't tell that they're cloning, and it may lead to situations where the caller has owned data which they pass by reference, causing it to be cloned unnecessarily.

Reply:
Good catch. There may be more instances of this, deferred to https://github.com/googlefonts/fontc/issues/363.